roles/ceph-docker-common/tasks/pre_requisites/prerequisite_docker.yml

```diff
@@ -123,27 +137,30 @@
     with_pkg
 
 # NOTE (leseb): for version 1.1.0 because https://github.com/ansible/ansible-modules-core/issues/1227
-- name: install docker-py
+- name: install docker-py via pip for every distribution except rhel (ansible < 2.1.0.0)
   pip:
     name: docker-py
     version: 1.1.0
   tags:
     with_pkg
-  when: ansible_version['full'] | version_compare('2.1.0.0', '<')
+  when:
+    - ansible_version['full'] | version_compare('2.1.0.0', '<')
```

Comment:
Should we care about ansible versions < 2.2.1 ? We only test ``ansible==2.2.1`` with our CI tests currently.